### PR TITLE
bt-360: Fix mobile

### LINKED ISSUE #360: Fix mobile
```patch
diff --git a/mobile/src/bundles/auth/screens/auth.tsx b/mobile/src/bundles/auth/screens/auth.tsx
index 125011d9..d6ded691 100644
--- a/mobile/src/bundles/auth/screens/auth.tsx
+++ b/mobile/src/bundles/auth/screens/auth.tsx
@@ -1,3 +1,4 @@
+// FIX: structure
 import React from 'react';
 
 import { actions as authActions } from '~/bundles/auth/store';
@@ -43,6 +44,8 @@ const Auth: React.FC = () => {
         },
         [dispatch],
     );
+
+    // FIX: Split to different screens. We dont need to mix logic
     const getScreen = (screen: string): React.ReactNode => {
         switch (screen) {
             case AuthScreenName.SIGN_IN: {
diff --git a/mobile/src/bundles/common/components/image-picker/image-picker.tsx b/mobile/src/bundles/common/components/image-picker/image-picker.tsx
index 1db679e7..7b42735f 100644
--- a/mobile/src/bundles/common/components/image-picker/image-picker.tsx
+++ b/mobile/src/bundles/common/components/image-picker/image-picker.tsx
@@ -1,8 +1,8 @@
 import React from 'react';
-import { PermissionsAndroid } from 'react-native';
-import { type StyleProp, type ViewStyle } from 'react-native';
-import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
-import { type ImagePickerResponse } from 'react-native-image-picker';
+import { PermissionsAndroid } from 'react-native'; // FIX: import
+import { type StyleProp, type ViewStyle } from 'react-native'; // FIX: import
+import { launchCamera, launchImageLibrary } from 'react-native-image-picker'; // FIX: import
+import { type ImagePickerResponse } from 'react-native-image-picker'; // FIX: import
 
 import {
     Button,
diff --git a/mobile/src/bundles/common/components/input/input.tsx b/mobile/src/bundles/common/components/input/input.tsx
index 6899a32f..5396d6a2 100644
--- a/mobile/src/bundles/common/components/input/input.tsx
+++ b/mobile/src/bundles/common/components/input/input.tsx
@@ -4,8 +4,8 @@ import {
     type FieldPath,
     type FieldValues,
 } from 'react-hook-form';
-import { type TextInputProps } from 'react-native';
-import { TextInput } from 'react-native';
+import { type TextInputProps } from 'react-native'; // FIX: import
+import { TextInput } from 'react-native'; // FIX: import
 
 import { Text, View } from '~/bundles/common/components/components';
 import { TextCategory } from '~/bundles/common/enums/enums';
diff --git a/mobile/src/bundles/common/components/modals/active-modal.tsx b/mobile/src/bundles/common/components/modals/active-modal.tsx
index 809448c6..4d9bcd37 100644
--- a/mobile/src/bundles/common/components/modals/active-modal.tsx
+++ b/mobile/src/bundles/common/components/modals/active-modal.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import React from 'react';
 
 import {
diff --git a/mobile/src/bundles/common/components/modals/modal.tsx b/mobile/src/bundles/common/components/modals/modal.tsx
index 57df3e45..02e88ab0 100644
--- a/mobile/src/bundles/common/components/modals/modal.tsx
+++ b/mobile/src/bundles/common/components/modals/modal.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import React from 'react';
 
 import {
diff --git a/mobile/src/bundles/common/helpers/split-array-in-half.ts b/mobile/src/bundles/common/helpers/split-array-in-half.ts
index a6090c88..46d61902 100644
--- a/mobile/src/bundles/common/helpers/split-array-in-half.ts
+++ b/mobile/src/bundles/common/helpers/split-array-in-half.ts
@@ -1,3 +1,4 @@
+// FIX: project structure
 const COLUMNS_NUMBER = 2;
 const START_INDEX = 0;
 
diff --git a/mobile/src/bundles/common/styles/global-styles.ts b/mobile/src/bundles/common/styles/global-styles.ts
index 050ec92b..ecf2da48 100644
--- a/mobile/src/bundles/common/styles/global-styles.ts
+++ b/mobile/src/bundles/common/styles/global-styles.ts
@@ -1,3 +1,4 @@
+// FIX: you can combine all styles in one file 'styles.ts'
 import { StyleSheet } from 'react-native';
 
 import { FLEX } from './flex';
diff --git a/mobile/src/bundles/talent/components/new-account-header/new-account-header.tsx b/mobile/src/bundles/talent/components/new-account-header/new-account-header.tsx
index 33a5c10c..3bb64805 100644
--- a/mobile/src/bundles/talent/components/new-account-header/new-account-header.tsx
+++ b/mobile/src/bundles/talent/components/new-account-header/new-account-header.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import React from 'react';
 
 import { Text, View } from '~/bundles/common/components/components';
diff --git a/mobile/src/bundles/talent/components/new-account-header/step-progress-bar.tsx b/mobile/src/bundles/talent/components/new-account-header/step-progress-bar.tsx
index 1792893b..aafdff89 100644
--- a/mobile/src/bundles/talent/components/new-account-header/step-progress-bar.tsx
+++ b/mobile/src/bundles/talent/components/new-account-header/step-progress-bar.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import React from 'react';
 
 import { View } from '~/bundles/common/components/components';
diff --git a/mobile/src/bundles/talent/components/new-account-header/step-track.tsx b/mobile/src/bundles/talent/components/new-account-header/step-track.tsx
index b237c578..ca6431a6 100644
--- a/mobile/src/bundles/talent/components/new-account-header/step-track.tsx
+++ b/mobile/src/bundles/talent/components/new-account-header/step-track.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import React from 'react';
 
 import { Text, View } from '~/bundles/common/components/components';
diff --git a/mobile/src/bundles/talent/components/profile-form/employment-types.tsx b/mobile/src/bundles/talent/components/profile-form/employment-types.tsx
index 1c114507..defa675d 100644
--- a/mobile/src/bundles/talent/components/profile-form/employment-types.tsx
+++ b/mobile/src/bundles/talent/components/profile-form/employment-types.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import React from 'react';
 import {
     type Control,
diff --git a/mobile/src/bundles/talent/components/profile-form/profile-form.tsx b/mobile/src/bundles/talent/components/profile-form/profile-form.tsx
index f512e61a..d476930f 100644
--- a/mobile/src/bundles/talent/components/profile-form/profile-form.tsx
+++ b/mobile/src/bundles/talent/components/profile-form/profile-form.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import { type NavigationProp } from '@react-navigation/native';
 import React from 'react';
 
diff --git a/mobile/src/bundles/talent/screens/cv-and-contacts.tsx b/mobile/src/bundles/talent/screens/cv-and-contacts.tsx
index 7095d439..4afa44fe 100644
--- a/mobile/src/bundles/talent/screens/cv-and-contacts.tsx
+++ b/mobile/src/bundles/talent/screens/cv-and-contacts.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import React from 'react';
 
 import { View } from '~/bundles/common/components/components';
diff --git a/mobile/src/bundles/talent/screens/mail.tsx b/mobile/src/bundles/talent/screens/mail.tsx
index 93be4f7a..34f1224e 100644
--- a/mobile/src/bundles/talent/screens/mail.tsx
+++ b/mobile/src/bundles/talent/screens/mail.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import React from 'react';
 
 import { Text } from '~/bundles/common/components/components';
diff --git a/mobile/src/bundles/talent/screens/preview.tsx b/mobile/src/bundles/talent/screens/preview.tsx
index d9dcf26c..98dce9d7 100644
--- a/mobile/src/bundles/talent/screens/preview.tsx
+++ b/mobile/src/bundles/talent/screens/preview.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import React from 'react';
 
 import { Text } from '~/bundles/common/components/components';
diff --git a/mobile/src/bundles/talent/screens/profile.tsx b/mobile/src/bundles/talent/screens/profile.tsx
index 7e41472f..dc0c5567 100644
--- a/mobile/src/bundles/talent/screens/profile.tsx
+++ b/mobile/src/bundles/talent/screens/profile.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import React from 'react';
 
 import { View } from '~/bundles/common/components/components';
diff --git a/mobile/src/bundles/talent/screens/skills-and-projects.tsx b/mobile/src/bundles/talent/screens/skills-and-projects.tsx
index 7df37c93..34cfeeb4 100644
--- a/mobile/src/bundles/talent/screens/skills-and-projects.tsx
+++ b/mobile/src/bundles/talent/screens/skills-and-projects.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import React from 'react';
 
 import { Text } from '~/bundles/common/components/components';
diff --git a/mobile/src/bundles/talent/screens/talent-profile.tsx b/mobile/src/bundles/talent/screens/talent-profile.tsx
index f6132ca1..e9fc0717 100644
--- a/mobile/src/bundles/talent/screens/talent-profile.tsx
+++ b/mobile/src/bundles/talent/screens/talent-profile.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import React from 'react';
 
 import { Text } from '~/bundles/common/components/components';
diff --git a/mobile/src/bundles/users/types/components/avatar-properties.ts b/mobile/src/bundles/users/types/components/avatar-properties.ts
index 07bd37e1..ae0496f9 100644
--- a/mobile/src/bundles/users/types/components/avatar-properties.ts
+++ b/mobile/src/bundles/users/types/components/avatar-properties.ts
@@ -1,3 +1,4 @@
+// FIX: project structure. Create 'types.ts'
 import { type ValueOf } from 'shared/build/types/value-of.type';
 
 import { type AvatarType } from '~/bundles/common/enums/ui/avatar-type.enum';
diff --git a/mobile/src/navigations/bottom-tab-navigator/employer-bottom-tab.tsx b/mobile/src/navigations/bottom-tab-navigator/employer-bottom-tab.tsx
index afb00b0a..1e02b2e3 100644
--- a/mobile/src/navigations/bottom-tab-navigator/employer-bottom-tab.tsx
+++ b/mobile/src/navigations/bottom-tab-navigator/employer-bottom-tab.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import React from 'react';
 import Icon from 'react-native-vector-icons/MaterialIcons';
diff --git a/mobile/src/navigations/bottom-tab-navigator/talent-bottom-tab.tsx b/mobile/src/navigations/bottom-tab-navigator/talent-bottom-tab.tsx
index 345c4852..dc267513 100644
--- a/mobile/src/navigations/bottom-tab-navigator/talent-bottom-tab.tsx
+++ b/mobile/src/navigations/bottom-tab-navigator/talent-bottom-tab.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import React from 'react';
 import Icon from 'react-native-vector-icons/MaterialIcons';
diff --git a/mobile/src/navigations/onboarding-navigator/talent-onboarding.tsx b/mobile/src/navigations/onboarding-navigator/talent-onboarding.tsx
index 3ba51e1a..cd3fe451 100644
--- a/mobile/src/navigations/onboarding-navigator/talent-onboarding.tsx
+++ b/mobile/src/navigations/onboarding-navigator/talent-onboarding.tsx
@@ -1,3 +1,4 @@
+// FIX: project structure
 import {
     createDrawerNavigator,
     type DrawerContentComponentProps,

```

### PR DESCRIPTION
**Pull request by anogire**

### CODE REVIEW

File: mobile/src/bundles/auth/enums/enums.ts [hunk 1]
```diff
@@ -1 +1 @@
-export { AuthApiPath } from 'shared/build/index';
+export { AuthApiPath, UserRole } from 'shared/build/index.js';
```
Comment (mskmee) [inline mobile/src/bundles/auth/enums/enums.ts]: do we need file extension here?

File: mobile/src/bundles/auth/types/types.ts [hunk 2]
```diff
@@ -1,9 +1,7 @@
 export {
     type UserFindResponseDto,
-    type UserGetAllItemResponseDto,
-    type UserGetAllResponseDto,
     type UserSignInRequestDto,
     type UserSignInResponseDto,
     type UserSignUpRequestDto,
     type UserSignUpResponseDto,
-} from 'shared/build/index';
+} from 'shared/build/index.js';
```
Comment (mskmee) [inline mobile/src/bundles/auth/types/types.ts:7]: Same here

File: mobile/src/bundles/auth/validation-schemas/validation-schemas.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
 export {
     userSignInValidationSchema,
     userSignUpValidationSchema,
-} from 'shared/build/index';
+} from 'shared/build/index.js';
```
Comment (mskmee) [inline mobile/src/bundles/auth/validation-schemas/validation-schemas.ts:4]: and here)

File: mobile/src/bundles/common/components/autocomplete-selectors/autocomplete-multi-selector/autocomplete-multi-selector.tsx [hunk 4]
```diff
@@ -23,8 +18,13 @@ import {
...
 } from '~/bundles/common/hooks/hooks';
 import { globalStyles } from '~/bundles/common/styles/styles';
+import {
+    type Control,
+    type FieldPath,
+    type FieldValues,
+} from '~/bundles/common/types/types';
 
-import { styles } from './styles';
+import { styles } from '../styles';
```
Comment (mskmee) [inline mobile/src/bundles/common/components/autocomplete-selectors/autocomplete-multi-selector/autocomplete-multi-selector.tsx:26]: Maybe better to use absolute path here

File: mobile/src/bundles/common/components/autocomplete-selectors/autocomplete-selector/autocomplete-selector.tsx [hunk 5]
```diff
@@ -22,9 +17,14 @@ import {
     useVisibility,
 } from '~/bundles/common/hooks/hooks';
 import { globalStyles } from '~/bundles/common/styles/styles';
+import {
+    type Control,
+    type FieldPath,
+    type FieldValues,
+} from '~/bundles/common/types/types';
 
+import { styles } from '../styles';
```
Comment (mskmee) [inline mobile/src/bundles/common/components/autocomplete-selectors/autocomplete-selector/autocomplete-selector.tsx:25]: Same here

File: mobile/src/bundles/common/components/modals/active-modal/active-modal.tsx [hunk 6]
```diff
@@ -10,7 +10,7 @@ import {
 import { ButtonType, TextCategory } from '~/bundles/common/enums/enums';
 import { globalStyles } from '~/bundles/common/styles/styles';
 
-import { styles } from './styles';
+import { styles } from '../styles';
```
Comment (mskmee) [inline mobile/src/bundles/common/components/modals/active-modal/active-modal.tsx:13]: Maybe its better to use absolute path here

File: mobile/src/bundles/common/components/modals/modal/modal.tsx [hunk 7]
```diff
@@ -7,7 +7,7 @@ import {
 } from '~/bundles/common/components/components';
 import { globalStyles } from '~/bundles/common/styles/styles';
 
-import { styles } from './styles';
+import { styles } from '../styles';
```
Comment (mskmee) [inline mobile/src/bundles/common/components/modals/modal/modal.tsx:10]: Maybe its better to use absolute path here

File: mobile/src/bundles/common/enums/enums.ts [hunk 8]
```diff
@@ -22,6 +22,9 @@ export {
     ApiPath,
     AppEnvironment,
     ContentType,
+    EnglishLevel,
     ErrorMessages,
+    NotConsidered,
+    PreferredLanguages,
     ServerErrorType,
-} from 'shared/build/index';
+} from 'shared/build/index.js';
```
Comment (mskmee) [inline mobile/src/bundles/common/enums/enums.ts:35]: file extension

File: mobile/src/bundles/common/helpers/helpers.ts [hunk 9]
```diff
@@ -1,3 +1,3 @@
 export { getErrorMessage } from './get-error-message/get-error-message';
-export { splitArrayInHalf } from './split-array-in-half';
-export { configureString, getAvatarInitials } from 'shared/build/index';
+export { splitArrayInHalf } from './split-array-in-half/split-array-in-half';
+export { getAvatarInitials } from 'shared/build/index.js';
```
Comment (mskmee) [inline mobile/src/bundles/common/helpers/helpers.ts:7]: file extension

File: mobile/src/bundles/common/types/types.ts [hunk 10]
```diff
@@ -1,17 +1,56 @@
...
+    type RadioGroupProps,
+} from 'react-native-radio-buttons-group';
+export {
+    type BsaBadgeStepBadgesTitle,
     type ServerErrorDetail,
     type ServerErrorResponse,
     type ValidationSchema,
     type ValueOf,
-} from 'shared/build/index';
+} from 'shared/build/index.js';
```
Comment (mskmee) [inline mobile/src/bundles/common/types/types.ts:57]: file extension

File: mobile/src/bundles/employer/enums/enums.ts [hunk 11]
```diff
@@ -1 +1,6 @@
-export { CountryList, EmploymentType, JobTitle } from 'shared/build/index';
+export {
+    BsaBadgeStepBadgesTitle,
+    CountryList,
+    EmploymentType,
+    JobTitle,
+} from 'shared/build/index.js';
```
Comment (mskmee) [inline mobile/src/bundles/employer/enums/enums.ts]: file extension

File: mobile/src/bundles/talent/components/new-account-header/step-progress-bar/step-progress-bar.tsx [hunk 12]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { View } from '~/bundles/common/components/components';
 import { globalStyles } from '~/bundles/common/styles/styles';
 
-import { styles } from './styles';
+import { styles } from '../styles';
```
Comment (mskmee) [inline mobile/src/bundles/talent/components/new-account-header/step-progress-bar/step-progress-bar.tsx:6]: path

File: mobile/src/bundles/talent/components/new-account-header/step-track/step-track.tsx [hunk 13]
```diff
@@ -4,7 +4,7 @@ import { Text, View } from '~/bundles/common/components/components';
 import { TextCategory } from '~/bundles/common/enums/enums';
 import { globalStyles } from '~/bundles/common/styles/styles';
 
-import { styles } from './styles';
+import { styles } from '../styles';
```
Comment (mskmee) [inline mobile/src/bundles/talent/components/new-account-header/step-track/step-track.tsx:7]: path

File: mobile/src/bundles/talent/helpers/check-if-file-size-valid/check-if-file-size-valid.ts [hunk 14]
```diff
@@ -1,4 +1,4 @@
-import { MAX_FILE_SIZE } from './constants/constants';
+import { MAX_FILE_SIZE } from '../constants/constants';
```
Comment (mskmee) [inline mobile/src/bundles/talent/helpers/check-if-file-size-valid/check-if-file-size-valid.ts:1]: path

File: mobile/src/bundles/talent/helpers/check-if-image-type-valid/check-if-image-type-valid.ts [hunk 15]
```diff
@@ -1,4 +1,4 @@
-import { ACCEPTED_PHOTO_TYPES } from './constants/constants';
+import { ACCEPTED_PHOTO_TYPES } from '../constants/constants';
```
Comment (mskmee) [inline mobile/src/bundles/talent/helpers/check-if-image-type-valid/check-if-image-type-valid.ts:1]: path

File: mobile/src/bundles/users/users.ts [hunk 16]
```diff
@@ -14,4 +14,4 @@ export { userApi };
 export {
     type UserGetAllItemResponseDto,
     type UserGetAllResponseDto,
-} from 'shared/build/index';
+} from 'shared/build/index.js';
```
Comment (mskmee) [inline mobile/src/bundles/users/users.ts]: extension

File: mobile/src/framework/config/types/config.type.ts [hunk 17]
```diff
@@ -1,4 +1,4 @@
-import { type Config as ConfigBase } from 'shared/build/index';
+import { type Config as ConfigBase } from 'shared/build/index.js';
```
Comment (mskmee) [inline mobile/src/framework/config/types/config.type.ts:1]: extension

File: mobile/src/framework/http/enums/enums.ts [hunk 18]
```diff
@@ -1 +1 @@
-export { HttpCode, HttpHeader } from 'shared/build/index';
+export { HttpCode, HttpHeader } from 'shared/build/index.js';
```
Comment (mskmee) [inline mobile/src/framework/http/enums/enums.ts:1]: extension

File: mobile/src/framework/http/exceptions/http-error.exception.ts [hunk 19]
```diff
@@ -1,4 +1,4 @@
-import { HttpError as LibraryHttpError } from 'shared/build/index';
+import { HttpError as LibraryHttpError } from 'shared/build/index.js';
```
Comment (mskmee) [inline mobile/src/framework/http/exceptions/http-error.exception.ts:1]: extension

File: mobile/src/framework/http/types/types.ts [hunk 20]
```diff
@@ -1,2 +1 @@
-export { type HttpOptions } from 'shared/build/index';
-export { type Http } from 'shared/build/index';
+export { type Http, type HttpOptions } from 'shared/build/index.js';
```
Comment (mskmee) [inline mobile/src/framework/http/types/types.ts:1]: extension

File: mobile/src/framework/storage/types/types.ts [hunk 21]
```diff
@@ -1 +1 @@
-export { type Storage } from 'shared/build/index';
+export { type Storage } from 'shared/build/index.js';
```
Comment (mskmee) [inline mobile/src/framework/storage/types/types.ts:1]: extension

File: mobile/src/helpers/helpers.ts [hunk 22]
```diff
@@ -1 +1 @@
-export { configureString } from 'shared/build/index';
+export { configureString } from 'shared/build/index.js';
```
Comment (mskmee) [inline mobile/src/helpers/helpers.ts:1]: extension

File: mobile/src/navigations/bottom-tab-navigator/employer-bottom-tab/employer-bottom-tab.tsx [hunk 23]
```diff
@@ -13,7 +15,7 @@ import {
     EmployerProfile,
 } from '~/bundles/employer/screens/screens';
 
-import { bottomTabStyles } from './styles';
+import { bottomTabStyles } from '../styles';
```
Comment (mskmee) [inline mobile/src/navigations/bottom-tab-navigator/employer-bottom-tab/employer-bottom-tab.tsx:16]: path

File: mobile/src/navigations/onboarding-navigator/talent-onboarding-navigator/components/steps/steps.tsx [hunk 24]
```diff
@@ -21,14 +18,19 @@ import {
...
+} from '~/bundles/common/hooks/hooks';
 import { globalStyles } from '~/bundles/common/styles/styles';
 import {
+    type DrawerContentComponentProps,
     type TalentOnboardingRouteProperties,
     type ValueOf,
 } from '~/bundles/common/types/types';
 
-import { Step } from '../components';
+import { Step } from '../step/step';
```
Comment (mskmee) [inline mobile/src/navigations/onboarding-navigator/talent-onboarding-navigator/components/steps/steps.tsx]: path

File: mobile/src/bundles/auth/enums/enums.ts (hunk 1 above)
Comment (anogire) [reply mobile/src/bundles/auth/enums/enums.ts]: I think yes, for files from shared/build folder

File: mobile/src/bundles/common/components/autocomplete-selectors/autocomplete-multi-selector/autocomplete-multi-selector.tsx (hunk 4 above)
Comment (anogire) [reply mobile/src/bundles/common/components/autocomplete-selectors/autocomplete-multi-selector/autocomplete-multi-selector.tsx:26]: I left the following path for those files with the styles that are in the same folder

File: mobile/src/bundles/auth/components/auth-wrapper/styles.ts [hunk 25]
```diff
@@ -1,5 +1,4 @@
-import { StyleSheet } from 'react-native';
-
+import { StyleSheet } from '~/bundles/common/components/components';
```
Comment (katerynakostikova) [inline mobile/src/bundles/auth/components/auth-wrapper/styles.ts]: I think it's better to get if from styles folder

File: mobile/src/bundles/common/components/autocomplete-selectors/autocomplete-multi-selector/autocomplete-multi-selector.tsx [hunk 26]
```diff
@@ -1,16 +1,11 @@
...
-import {
-    type Control,
-    type FieldPath,
-    type FieldValues,
-} from 'react-hook-form';
-import { TextInput } from 'react-native';
-import Animated from 'react-native-reanimated';
 
 import {
+    Animated,
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/components/autocomplete-selectors/autocomplete-multi-selector/autocomplete-multi-selector.tsx]: Also styles

File: mobile/src/bundles/common/components/image-picker/image-picker.tsx [hunk 27]
```diff
@@ -1,12 +1,11 @@
 import React from 'react';
-import { PermissionsAndroid } from 'react-native';
-import { type StyleProp, type ViewStyle } from 'react-native';
-import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
-import { type ImagePickerResponse } from 'react-native-image-picker';
 
 import {
     Button,
+    launchCamera,
+    launchImageLibrary,
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/components/image-picker/image-picker.tsx]: helpers

File: mobile/src/bundles/common/components/image-picker/image-picker.tsx [hunk 28]
```diff
@@ -1,12 +1,11 @@
...
     Modal,
+    PermissionsAndroid,
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/components/image-picker/image-picker.tsx]: constants maybe

File: mobile/src/bundles/employer/screens/candidates-filter/candidates-filter.tsx [hunk 29]
```diff
@@ -1,29 +1,22 @@
 import React from 'react';
-import Icon from 'react-native-vector-icons/MaterialIcons';
 
 import {
     AutocompleteMultiSelector,
     Button,
     CheckboxGroup,
     Divider,
     FormField,
+    Icon,
```
Comment (katerynakostikova) [inline mobile/src/bundles/employer/screens/candidates-filter/candidates-filter.tsx]: If it's material icon (not community), let's name it MaterialIcon

File: mobile/src/navigations/auth-navigator/auth-navigator.tsx [hunk 30]
```diff
@@ -1,12 +1,12 @@
-import {
-    createNativeStackNavigator,
-    type NativeStackNavigationOptions,
-} from '@react-navigation/native-stack';
 import React from 'react';
 
-import { Auth } from '~/bundles/auth/screens/auth';
+import { SignIn, SignUp } from '~/bundles/auth/screens/screens';
+import { createNativeStackNavigator } from '~/bundles/common/components/components';
```
Comment (katerynakostikova) [inline mobile/src/navigations/auth-navigator/auth-navigator.tsx]: helpers

File: mobile/src/navigations/bottom-tab-navigator/talent-bottom-tab/talent-bottom-tab.tsx [hunk 31]
```diff
@@ -1,15 +1,17 @@
...
+} from '~/bundles/common/components/components';
 import {
     IconName,
     TalentBottomTabScreenName,
 } from '~/bundles/common/enums/enums';
 import { type TalentBottomTabNavigationParameterList } from '~/bundles/common/types/types';
 import { Mail, TalentProfile } from '~/bundles/talent/screens/screens';
 
-import { bottomTabStyles } from './styles';
+import { bottomTabStyles } from '../styles';
```
Comment (katerynakostikova) [inline mobile/src/navigations/bottom-tab-navigator/talent-bottom-tab/talent-bottom-tab.tsx:12]: Why also a lot of components have styles on upper level?

File: mobile/src/navigations/onboarding-navigator/talent-onboarding-navigator/components/steps/steps.tsx [hunk 32]
```diff
@@ -1,15 +1,12 @@
...
-import Animated, { interpolate } from 'react-native-reanimated';
-import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
 
 import { logout } from '~/bundles/auth/store/actions';
 import {
+    Animated,
     Button,
+    CommunityIcon,
+    DrawerContentScrollView,
+    interpolate,
```
Comment (katerynakostikova) [inline mobile/src/navigations/onboarding-navigator/talent-onboarding-navigator/components/steps/steps.tsx]: helpers, or styles

File: mobile/src/bundles/auth/components/auth-wrapper/styles.ts (hunk 25 above)
Comment (anogire) [reply mobile/src/bundles/auth/components/auth-wrapper/styles.ts:1]: You mean left import { StyleSheet } from 'react-native'; in styles.ts?

File: mobile/src/navigations/bottom-tab-navigator/talent-bottom-tab/talent-bottom-tab.tsx (hunk 31 above)
Comment (anogire) [reply mobile/src/navigations/bottom-tab-navigator/talent-bottom-tab/talent-bottom-tab.tsx:12]: Here we have: 
```
bottom-tab-navigator
-----employer-bottom-tab -> employer-bottom-tab.tsx
-----talent-bottom-tab -> talent-bottom-tab.tsx
bottom-tab-navigator.ts
styles.ts - the same for employer-bottom-tab.tsx and talent-bottom-tab.tsx
```

For these situations I left one file styles.ts and its path: ../styles
Or should I create separate styles.ts for mployer-bottom-tab and talent-bottom-tab folders?

Comment (katerynakostikova) [reply mobile/src/navigations/bottom-tab-navigator/talent-bottom-tab/talent-bottom-tab.tsx:12]: Ok, let's leave it like this

**Review by katerynakostikova — APPROVED** [review]

**Review by mskmee — APPROVED** [review]

**Pull request merged**